### DISCUSSION TITLE
Move fast and break less with GitHub Mobile [DEV1929D]

### DISCUSSION THREAD

**Discussion by Iron-Ham** [community/community#74668] [Mobile]
**Speaker:** @Iron-Ham 
**Social:** You can follow me on [Mastodon](https://hachyderm.io/@IronHam), [LinkedIn](https://www.linkedin.com/in/heshamsalman/) or my [website](https://sundayswift.com/)

> In a world where many of us work asynchronously across time and physical space, we also don't have to limit ourselves by working at a desk. Whether you're a developer, a product manager, or an engineering manager, come learn how GitHub Mobile can help your team unblock work and tackle tasks on the go.

Thank you for joining my session! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll do my best to answer.

**Comment by Ridvan8899** [comment]
مرحبا

**Comment by Ridvan8899** [comment]
يا سيدي انا مستخدم جديد هنا
لكن تفاجات يوجد تطبيق هذا في هاتفي
منذ اسبوع تعرض محافظي العملات الرقمية للسرقه 
هل هذا البرنامج لهم ؟!!
